client/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,130 @@
+
+// Styles
+import useStyles from './useStyles';
+
+function LoggedInBar({ classes, loggedInUser }: any): JSX.Element {
+  return (
+    <Box>
+      <NavLink to="/dashboard" className={classes.link}>
+        <Button className={classes.menuBarButton}>
+          <Badge color="primary" variant="dot">
+            <Typography variant="h6" color="textPrimary" className={`${classes.menuBarText} ${classes.mobileView}`}>
+              Notifications
+            </Typography>
+            <Notifications className={`${classes.desktopView}`} />
+          </Badge>
+        </Button>
+      </NavLink>
+      {loggedInUser.isSitter ? (
+        <NavLink to="/my-jobs" className={`${classes.link} ${classes.mobileView}`}>
+          <Button className={classes.menuBarButton}>
+            <Typography variant="h6" color="textPrimary" className={classes.menuBarText}>
+              My Jobs
+            </Typography>
+          </Button>
+        </NavLink>
+      ) : (
+        <NavLink to="/my-sitters" className={`${classes.link} ${classes.mobileView}`}>
+          <Button className={classes.menuBarButton}>
+            <Typography variant="h6" color="textPrimary" className={classes.menuBarText}>
+              My Sitters
+            </Typography>
+          </Button>
+        </NavLink>
+      )}
+
+      <NavLink to="/messages" className={`${classes.link} ${classes.mobileView}`}>
+        <Button className={classes.menuBarButton}>
+          <Badge color="primary" variant="dot">
+            <Typography variant="h6" color="textPrimary" className={classes.menuBarText}>
+              Messages
+            </Typography>
+          </Badge>
+        </Button>
+      </NavLink>
+      <AuthMenu />
+    </Box>
+  );
+}
+
+function LoginBar({ classes }: any): JSX.Element {
+  return (
+    <Box>
+      <MaterialLink href="/dashboard" color="textPrimary" className={classes.sitterText} underline="always">
+        BECOME A DOG SITTER
+      </MaterialLink>
+      <NavLink to="/login" className={classes.link}>
+        <Button variant="outlined" size="large" color="secondary" className={classes.menuBarButton}>
+          <Typography variant="h6" color="inherit" style={{ fontWeight: 'bold' }}>
+            Login
+          </Typography>
+        </Button>
+      </NavLink>
+      <NavLink to="/signup" className={classes.link}>
+        <Button variant="contained" size="large" color="secondary" className={classes.menuBarButton}>
+          <Typography variant="h6" color="inherit" className={classes.menuBarText}>
+            SignUp
+          </Typography>
+        </Button>
+      </NavLink>
+    </Box>
+  );
+}
+
+export default function NavBar(): JSX.Element {
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+  const classes = useStyles();
+
+  useEffect(() => {
+    initSocket();
+  }, [initSocket]);
+
+  if (loggedInUser === undefined) return <CircularProgress />;
+
+  return (
+    <>
+      <CssBaseline />
```

Comment:
Please remove the <CssBaseline /> here as it is a global reset component and only needs to be included once, ideally in the 'App.tsx'. If you still intend to use a baseline for the children, look into the usage of a 'ScopedCssBaseline'

Ref: https://mui.com/api/scoped-css-baseline/